gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -1185,9 +1185,10 @@ def GetThac0 (Class, Level):
 	return Thac0Table.GetValue (Class, str (Level))
 
 #apparently the original code doesn't follow profsmax/profs tables
-def HasGainedWeapProf (pc, currProf, currLevel, Class):
+def HasGainedWeapProf (pc, currProf, currLevel):
 	#only fighters gain weapon proficiencies
-	if Class!="FIGHTER":
+ 	Class = GUICommon.GetClassRowName(pc).split("_")
+	if Class[0]!="FIGHTER":
```

Comment:
Something cleaner like:
```
	Class = GUICommon.GetClassRowName (pc)
	if not "FIGHTER" in Class:
```

Reply:
That is way better and works just as well in practice, done.